media/static-bundles.json

```diff
@@ -1595,10 +1595,17 @@
     {
       "files": [
         "js/base/send-to-device.js",
+        "js/base/send-yourself.js",
         "js/firefox/whatsnew/whatsnew.js"
       ],
       "name": "firefox_whatsnew"
     },
+    {
+      "files": [
+        "js/base/send-yourself.js"
```

Comment:
You're loading the `send-yourself.js` script here, but you're missing JS that initialises the form:

```javascript
    var form = new Mozilla.SendYourself('download-firefox-rocket');
    form.init();
```

You'll likely need to create a new JS file like `whatsnew-id.js` where this gets initialized.

I noticed that the page header also seems broken, with the mozilla logo bleeding into the page:

![screenshot_2019-01-17 firefox lite](https://user-images.githubusercontent.com/400117/51307880-b04f7980-1a38-11e9-8dd2-8af0c6c8f538.png)

You'll likely need to include this snippet of JS like on other whatsnew pages that shows the up-to-date message (although the logo probably shouldn't look misaligned even if this message is not displayed):

https://github.com/mozilla/bedrock/blob/master/media/js/firefox/whatsnew/whatsnew-63.js#L10-L19